charts/spire/README.md

```diff
@@ -127,8 +132,3 @@ Kubernetes: `>=1.21.0-0`
 | waitForIt.image.registry | string | `"gcr.io"` |  |
 | waitForIt.image.repository | string | `"spiffe-io/wait-for-it"` |  |
 | waitForIt.image.version | string | `""` |  |
-| workloadRegistrar.image.pullPolicy | string | `"IfNotPresent"` |  |
```

Comment:
Maybe we can make one release by deprecating it.

e.g. enabled: false

and doing a enabled: true for the new controller.